src/common/cdomains/floatOps/stubs.c

```diff
@@ -101,3 +114,8 @@ CAMLprim value smallest_float(value unit)
 {
     return caml_copy_double(FLT_MIN);
 }
+
+CAMLprim value pi(value unit)
+{
+    return caml_copy_double(M_PI);
```

Comment:
Why do we need only the double version of `pi` and not a float version like we have for the unions?

Reply:
We found that their is implicit conversion from float to double and these stubs are only used for the `CFloat` module. The `CDouble` module can instead use the respective functions from the OCaml Float module (since OCaml Floats correspond to C doubles and we do not need to set the rounding mode here).